src/pages/notFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './NotFoundPage.scss';
```

Comment:
We agreed to use styles like this:
```suggestion
import classes from './NotFoundPage.module.scss';
```